assemblies/features/base/src/main/filtered-resources/resources/bin/client

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
(although bash may not be on all platforms, eg. restricted ones for containers, perhaps best just get rid of the bashisms!)